src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task {
+    public String at;
+    public Event(String title, String at) {
+        super(title);
+        this.at = at;
+    }
+
+    public String getTitle() {
+        return this.getType() + this.getStatus() + " " + title + " " + this.getAtString();
+    }
```

Comment:
Hi, is this considered unnecessary use of this with fields? Correct me if I am wrong.